src/lib/fetch.ts

```diff
@@ -162,6 +162,10 @@ export function _generateLinkResponse(data: any): GenerateLinkResponse {
   }
 }
 
+export function _noResolveJsonResponse(data: any): Response {
```

Comment:
we just needed to return the response object if `noResolveJson` is passed it because we want the whole response and not the body

Reply:
There is an option to pass in a transformer which is just a fancy word here for a callback function. The callback function will be used inside `_request` here: `return options?.xform ? options?.xform(data) : { data: { ...data }, error: null }`